packages/design-system-scripts/gulp/docs/webpack/index.js

```diff
@@ -7,19 +7,19 @@
  */
 const path = require('path');
 const util = require('util');
-const webpack = util.promisify(require('webpack'));
```

Comment:
@pwolfert I sunk a lot of time into this bug where webpack would work fine statically, but it would throw this undefined error for webpack static. I eventually found out that removing the promisify for the webpack server did the trick, but I don't really understand why because I was using `await` for both anyway?

Reply:
Hmm, I'm not sure why that would be. I'm sorry that it caused you so much grief though